packages/backend/src/database/knexfile.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
```

Comment:
I know that you used what i had on my draft PR, but upon second thought, I now think that the knex config and migrations should be moved out of src.

Let's move the knexfile.js in packages/backend, at the same level with src.
Also move the migrations directory in packges/backend at the same level with src.